test/IECore/IndexedIO.py

```diff
@@ -102,7 +102,7 @@ def testSaveWriteObjects(self):
 		self.assertEqual( txt, IECore.Object.load( f2, "obj1" ) )
 		self.assertEqual( txt, IECore.Object.load( f2, "obj2" ) )
 
-	@unittest.skipIf( IECore.isDebug(), "Skip performance testing in debug builds" )
+	@unittest.skipUnless( os.environ.get("IE_PERFORMANCE_TEST", False), "'IE_PERFORMANCE_TEST' env var not set" )
```

Comment:
@johnhaddon  You mentioned I should change this to ```IECORE_PERFORMANCE_TEST``` or was it ```CORTEX_PERFORMANCE_TEST```? 

Damn my failing memory.

Reply:
I think I hedged my bets and mentioned both. We seem to have a mishmash of both prefixes for other environment variables, but I'd perhaps tend towards `CORTEX_` if we intend this to cover other modules in the future...

Reply:
updated to `CORTEX_PERFORMANCE_TEST` in 9821b43